modules/COLL-01_Exploring-a-repository.yml

```diff
@@ -85,9 +89,13 @@ screens:
               id: COLL-01-qq4-o1
               response: "Good job! @crichID was the first person to comment on issue #2."
               correct: true
-            - value: # Null means it matches anything else - please make this the last option
+            - value: /hubyeti/i
```

Comment:
We are saying or, if its the class then instructor created the comment. If
it is self serve, hubyeti does it, so should accept either until there is a
better way to separate the two event types.

On Mon, Jul 27, 2015, 9:10 AM Cynthia Rich <notifications@github.com> wrote:

> In modules/COLL-01_Exploring-a-repository.yml
> <https://github.com/certify/curriculum-github/pull/107#discussion_r35533121>
> :
>
> > @@ -85,9 +89,13 @@ screens:
> >                id: COLL-01-qq4-o1
> >                response: "Good job! @crichID was the first person to comment on issue #2."
> >                correct: true
> > -            - value: # Null means it matches anything else - please make this the last option
> > +            - value: /hubyeti/i
>
> @kkemple <https://github.com/kkemple> same as above - are we leaving the
> option for either crichID or hubyeti to comment first?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/certify/curriculum-github/pull/107/files#r35533121>.
>